user/views.py

```diff
@@ -0,0 +1,47 @@
+from django.shortcuts import render, redirect
+
+from .forms import SignupForm, PasswordForm
+
+# Views for signing up
+def signup_view(request):
+    session_form_data = request.session.get('info_form_data')
+    form = SignupForm(session_form_data)
+    if request.method == "POST":
+        form = SignupForm(request.POST)
+        if form.is_valid():
+            request.session['info_form_data'] = request.POST
+            return redirect('user:create_password')
+    return render(request, 'user/signup.html', {'form':form})
```

Comment:
formを二回定義することはパフォーマンス上避けましょう
```suggestion
   if request.method == "POST":
        form = SignupForm(request.POST)
        if form.is_valid():
            request.session['info_form_data'] = request.POST
            return redirect('user:create_password')
   else:
       session_form_data = request.session.get('info_form_data')
       form = SignupForm(session_form_data)
   return render(request, 'user/signup.html', {'form':form})
```

Reply:
以下同

Reply:
おおなるほど、こう書けばよかったんですね。